content/docs/interface/example0/_index.md

```diff
@@ -1105,3 +1105,221 @@ write:
     data_product: human/outbreak/simulation_run-${{RUN_ID}}    
     version: 0.1.0
 ```
+
+## CodeRun with read globbing
+
+This example makes use of globbing in the `read:` block.
+
+First we need to populate your local registry with something to read:
+
+#### User written *config.yaml*
+
+```yaml
+run_metadata:
+  description: Register a file in the pipeline
+  local_data_registry_url: http://localhost:8000/api/
+  remote_data_registry_url: https://data.scrc.uk/api/
+  default_input_namespace: username
+  default_output_namespace: username
+  write_data_store: test_datastore/
```

Comment:
Shouldn't a `write_data_store` be an absolute path? You have it quite a bit in this PR, but I'm not confident I know what it's relative to...

Reply:
sure, that makes sense